integration/spark/src/test/scala/org/apache/carbondata/view/rewrite/MVCreateTestCase.scala

```diff
@@ -955,9 +955,9 @@ class MVCreateTestCase extends QueryTest with BeforeAndAfterAll {
     sql(" insert into mvtable1 select 'n4',12,12")
     sql("update mvtable1 set(name) = ('updatedName')").show()
     checkAnswer(sql("select count(*) from mvtable1 where name = 'updatedName'"),Seq(Row(4)))
+    sql(s"drop materialized view MV11")
```

Comment:
why this file change required, please revert if not required, i cant see any valid changes.

Reply:
Here, `drop materialized view` command is given after table drop and without `ifexists` check it fails.